src/worker/packages/server/src/entity/user-entity.ts

```diff
@@ -1,8 +1,12 @@
 import {EntitySchema} from "typeorm"
-import {User} from "shared/dist";
+import {Project, User} from "shared/dist";
```

Comment:
Seems like importing from `shared` directly doesn't work. Also importing from `shared/dist` seems to need a rebuild for every change. Can you please address this?